src/commonMain/kotlin/fr/acinq/lightning/NodeParams.kt

```diff
@@ -98,36 +103,27 @@ data class RecipientCltvExpiryParams(val min: CltvExpiryDelta, val max: CltvExpi
  * @param maxAcceptedHtlcs cap on the number of pending HTLCs in a channel: this lets us limit our exposure to HTLCs risk.
  * @param expiryDeltaBlocks cltv-expiry-delta used in our channel_update: since our channels are private and we don't relay payments, this will be basically ignored.
  * @param fulfillSafetyBeforeTimeoutBlocks number of blocks necessary to react to a malicious peer that doesn't acknowledge and sign our HTLC preimages.
- * @param checkHtlcTimeoutAfterStartupDelaySeconds delay in seconds before we check for timed out HTLCs in our channels after a wallet restart.
+ * @param checkHtlcTimeoutAfterStartupDelay delay in seconds before we check for timed out HTLCs in our channels after a wallet restart.
```

Comment:
nit: comment still stays that it is a number of seconds

Reply:
🦅 👁️

Reply:
Done in 7447a5cb.